plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Transform/Inline.hs

```diff
@@ -148,22 +149,22 @@ extendTerm ::
   -- | The substitution.
   S name uni fun a ->
   S name uni fun a
-extendTerm n clos s = s & subst . termEnv . unTermEnv %~ insertByName n clos
+extendTerm n clos s = s & subst . termEnv . unTermEnv %~ UMap.insertByName n clos
 
 lookupVarInfo ::
   (HasUnique name TermUnique) =>
   name ->
   S name uni fun a ->
   Maybe (VarInfo name uni fun a)
-lookupVarInfo n s = lookupName n $ s ^. vars
+lookupVarInfo n s = UMap.lookupName n $ s ^. vars
```

Comment:
Maybe we should implement https://hackage.haskell.org/package/lens-5.2.3/docs/Control-Lens-Combinators.html#t:At for `UniqueMap`?

Reply:
Same as https://github.com/IntersectMBO/plutus/pull/5815#discussion_r1514804028